lib/tesla/middleware/message_pack.ex

```diff
@@ -0,0 +1,160 @@
+defmodule Tesla.Middleware.MessagePack do
+  @moduledoc """
```

Comment:
```suggestion
if Code.ensure_loaded?(Msgpax) do
  defmodule Tesla.Middleware.MessagePack do
    @moduledoc """
```

Maybe we should ensure that `Msgpax` is part of the project

Reply:
done